devel/logcat-colorize/Portfile

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           github 1.0
+github.setup        carlonluca logcat-colorize 0.8.1 v
+github.tarball_from releases
+license             Apache-2
+categories          devel
+platforms           darwin
+maintainers         {@carlonluca} {gmail.com:carlon.luca} openmaintainer
```

Comment:
the GitHub handle and e-mail address need to be together withing brackets, like: 
`{@carlonluca gmail.com:carlon.luca}`